pkg/crawl/identify.go

```diff
@@ -340,11 +337,16 @@ func (i *Log4jIdentifier) printErrorFinding(message string, err error) {
 }
 
 func pathMatchesLog4JVersion(path string) (string, bool) {
+	filename := filenameFromPath(path)
+	return fileNameMatchesLog4jVersion(filename)
+}
+
+func filenameFromPath(path string) string {
```

Comment:
nit: Can we rename this to `filenameFromPathInsideArchive` so that semantically it's clear that we are only concerned with archive-specific path separators?